opal/mca/threads/argobots/configure.m4

```diff
@@ -60,6 +116,21 @@ AC_DEFUN([MCA_opal_threads_argobots_POST_CONFIG],[
            AC_DEFINE_UNQUOTED([MCA_threads_wait_sync_base_include_HEADER],
                               ["opal/mca/threads/argobots/threads_argobots_wait_sync.h"],
                               [Header to include for wait_sync implementation])
+           THREAD_CFLAGS="$TPKG_CFLAGS"
+           THREAD_FCFLAGS="$TPKG_FCFLAGS"
+           THREAD_CXXFLAGS="$TPKG_CXXFLAGS"
+           THREAD_CPPFLAGS="$TPKG_CPPFLAGS"
+           THREAD_CXXCPPFLAGS="$TPKG_CXXCPPFLAGS"
+           THREAD_LDFLAGS="$TPKG_LDFLAGS"
+           THREAD_LIBS="$TPKG_LIBS"
+           AC_SUBST(THREAD_CFLAGS)
+           AC_SUBST(THREAD_FCFLAGS)
+           AC_SUBST(THREAD_CXXFLAGS)
+           AC_SUBST(THREAD_CPPFLAGS)
```

Comment:
Shouldn't there be an `AC_SUBST(THREAD_CXXCPPFLAGS)` here?

Reply:
@hppritcha What about this one?

Reply:
I'm not sure THREAD_CXXCPPFLAGS is used in the top level configure.ac to generate the compiler wrappers, but I'll add it here in any case.

Reply:
@hppritcha  If it's not required, then no need to add it.  I just noticed that it's used directly above but not here.